docs/methods.md

```diff
@@ -192,3 +192,21 @@ if (kirby()->request()->is('POST')) {
     }
 }
 ```
+
+## withoutCSRF()
```

Comment:
I'd like to call this `disableCsrfProtection()` to make sure that you are disabling an important protection mechanism.